src/Microsoft.AspNetCore.SystemWebAdapters/Caching/Cache.cs

```diff
@@ -1,24 +1,152 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-namespace System.Web.Caching
+using System.Collections;
+using System.Runtime.Caching;
+
+namespace System.Web.Caching;
+
+public delegate void CacheItemRemovedCallback(string key, object value, CacheItemRemovedReason reason);
+public delegate void CacheItemUpdateCallback(string key, CacheItemUpdateReason reason, out object? expensiveObject, out CacheDependency? dependency, out DateTime absoluteExpiration, out TimeSpan slidingExpiration);
+
+public sealed class Cache : IEnumerable
```

Comment:
We can always add that later (there are additional properties/methods that are probably missing). We chose this based of higher usage of these. Since there's no analog, I'd prefer to not expose it unless we can rationalize what it may mean in ASP.NET Core.